File: problem_nine.py
#  Run rate for ODI mathch

target = int(input("Enter the target run:"))
current_run = int(input("Enter the Current run:"))
current_over = input("Enter the current over:")
def Runrate(run,over,target):
    Over = over.split(".")
    ball = int(Over[0])*6 + int(Over[1])
    bal_left = 300-ball
    runleft = target - run
    CurrR =(run/ball)*6
    ReqR = (runleft/bal_left)*6
    print("Current run rate:",round(CurrR,3))
    print("Requried run rate:",round(ReqR,3))

Runrate(current_run,current_over,target)

File: problem_nine.js
// Runrate  

var currR =225
var currO = "30.4"
var target = 331
function requiredRunrate(currR,currO,target){
    var run = target - currR
    var over = currO.split(".")
    var ball = parseInt(over[0]*6)+ parseInt(over[1])
    var rball = 300 - ball
    var currentRunrate =(currR/ball)*6
    var requiredRRate =(run/rball)*6
    console.log("Current Runrate:",currentRunrate.toFixed(2))
    console.log("Requried Runrate:", requiredRRate.toFixed(2))
}
requiredRunrate(currR,currO,target)




